src/Parsers/ParserCreateQuery.cpp

```diff
@@ -340,6 +340,7 @@ bool ParserStorage::parseImpl(Pos & pos, ASTPtr & node, Expected & expected)
     ParserKeyword s_partition_by("PARTITION BY");
     ParserKeyword s_primary_key("PRIMARY KEY");
     ParserKeyword s_order_by("ORDER BY");
+    ParserKeyword s_unique_key("UNIQUE KEY");
```

Comment:
We can specify `UNIQUE KEYS` different from `PRIMARY KEYS`, because usually we need several columns to be `PRIMARY KEYS` to speech up query performance, but `UNIQUE KEYS` can be just one or two columns.

Reply:
I Agree with having separate `UNIQUE KEY` for deduplication, it's better for power users of ClickHouse.

But it make sense to have some simplified, "MySQL" like behavior when PRIMARY KEY is uniq key as well.